test/unit/test_local_mode.py

```diff
@@ -204,7 +204,6 @@ def test_primitive_v2_with_not_accepted_options(self, primitive, backend):
         with warnings.catch_warnings(record=True) as warns:
             job = inst.run(**get_primitive_inputs(inst, backend=backend))
             _ = job.result()
-            self.assertEqual(len(warns), 1)
             self.assertIn("dynamical_decoupling", str(warns[0].message))
```

Comment:
I don't think it's the best idea to test for the number of warnings because we can't control the warnings that come from qiskit.